README.md

```diff
@@ -107,6 +107,16 @@ The following attributes are appended to a particular article within *articles.j
 
 <div float align=right><a href="#top">Back to Top</a></div>
 
+#### *components.json*
+
+**Numbering randomised questions**
+
+Two Handlebars Helpers are provided if you wish to have randomised questions named in sequential order. These can be used in the question's displayTitle in components.json.
```

Comment:
I don't think we should reference 'Handlebars Helpers' here as it's only going to be meaningful to a small subset of people reading this. Maybe change this to:
"If you need to display sequential question numbers within the component title when the questions are presented in a random order, the variables `{{question_number}}` and `{{question_count}}` can be used within the `displayTitle` text, for example: